pennylane/numpy/wrapper.py

```diff
@@ -71,6 +71,9 @@ def _wrapped(*args, **kwargs):
                 # and I'm not sure why. Using ``np.any`` works fine.
                 tensor_kwargs["requires_grad"] = _np.any([i.requires_grad for i in tensor_args])
 
+        if "is_input" in kwargs:
+            tensor_kwargs["is_input"] = kwargs.pop("is_input")

```

Comment:
Oh I think this was an accidental leftover from the `spectrum` function...

Reply:
Oops! That's for catching that, will remove.